bundles/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/DSMRPort.java

```diff
@@ -350,7 +425,8 @@ private boolean open() {
 
                 return false;
             } catch (UnsupportedCommOperationException ucoe) {
-                logger.error("Port is not suitable: " + portName, ucoe);
+                logger.error("Port does not support requested port settings "
+                        + "(invalid portsettings in openhab.cfg?): " + portName, ucoe);
```

Comment:
Please use parameterized logging instead of string concatenation.  Also, references to `openhab.cfg` might cause confusion as that config file is not normally used under openHAB 2.